### ISSUE TITLE
Add option for single quotes around module import strings with quotes: ["error", "backtick"]

### ISSUE THREAD

**Issue by janosh** [eslint/eslint#11241]
**What rule do you want to change?**

[`quotes`](https://eslint.org/docs/rules/quotes), specifically
```js
quotes: ["error", "backtick", { avoidEscape: true }]
```

**Does this change cause the rule to produce more or fewer warnings?**

Fewer

**How will the change be implemented? (New option, new default behavior, etc.)?**

New option, e.g.
```js
quotes: ["error", "backtick", { avoidEscape: true, import: "single" }]
```

**Example code that this change will affect:**

```jsx
import React from 'react'

const str = `some string`

export default () => <h1>{str}</h1>
```

**What the rule currently does for this code**

```jsx
import React from "react"  // note the change to double quotes here

const str = `some string`

export default () => <h1>{str}</h1>
```

**What the rule will do after it's changed?**

Keep the single quotes around `import` strings.

**Are you willing to submit a pull request to implement this change?**

Unable

**Comment by janosh** [comment]
@IanVS I guess this could be considered a follow-up to #3181 which fixed the problem of
```js
// .eslintrc.js
module.exports = {
  rules: {
     quotes: ["error", "backtick"]
}
```
requiring backticks even for ES6 imports, where they are invalid, but didn't add any way to specify whether to use single or double quotes instead, now simply defaulting to double quotes.
